lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -484,7 +497,10 @@ class QcFlagService {
                 }
             }
 
-            return qcFlagAdapter.toEntity(qcFlag);
+            const adaptedQcFlag = qcFlagAdapter.toEntity(qcFlag);
+            this._logger.infoMessage(`Delete QC flag with parameters: ${JSON.stringify(adaptedQcFlag)}`);
```

Comment:
You should not log the entire entity from the database. Please extract what you are interested in and log only that. 
Instead of `Delete`, please adapt with correct conjugation: `deleted` as  item was successfully removed already

Reply:
applied